Error: Cannot find module 'date-fns/format'

I added this library to my project for the stringified type, and it looks like a development build or something that relies on `date-fns` is trying to be imported, which I don't have in my project

```
$ ts-node src/index.ts
Error: Cannot find module 'date-fns/format'
Require stack:
- work_dir/node_modules/purify-ts-extra-codec/dist/purify-ts-extra-codec.cjs.development.js
- work_dir/node_modules/purify-ts-extra-codec/dist/index.js
- work_dir/path/to/index.ts
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (work_dir/node_modules/purify-ts-extra-codec/dist/purify-ts-extra-codec.cjs.development.js:8:34)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Module.require (internal/modules/cjs/loader.js:852:19)
```

I'm using `ts-node` inside a yarn project

node: v12.13.1
npm: 6.12.1
yarn: 1.3.2

--- a/README.md
+++ b/README.md
@@ -98,8 +98,6 @@ IntegerFromString.decode(1234); // Left("[error message]")
 
 ### Date Codec Module
 
-Date Codec Module requires [date-fns](https://date-fns.org/) peerDeps.
-
 #### DateFromAny
 
 Convert any date parsable object into Date.

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "purify-ts-extra-codec",
   "author": "airtoxin",
   "module": "dist/purify-ts-extra-codec.esm.js",
-  "version": "0.3.1",
+  "version": "0.3.2",
   "repository": "github:airtoxin/purify-ts-extra-codec",
   "license": "MIT",
   "main": "dist/index.js",

--- a/package.json
+++ b/package.json
@@ -23,7 +23,6 @@
     "prepare": "tsdx build"
   },
   "devDependencies": {
-    "date-fns": "^2.14.0",
     "eslint-plugin-prettier": "^3.1.3",
     "husky": "^4.2.5",
     "prettier": "^2.0.5",
@@ -32,9 +31,11 @@
     "tslib": "^2.0.0",
     "typescript": "^3.9.3"
   },
-  "peerDependencies": {
-    "purify-ts": "^0.15.1",
+  "dependencies": {
     "date-fns": "^2.14.0"
+  },
+  "peerDependencies": {
+    "purify-ts": "^0.15.1"
   },
   "husky": {
     "hooks": {

